libs/tendermint/cmd/tendermint/commands/run_node.go

```diff
@@ -176,6 +176,11 @@ func AddNodeFlags(cmd *cobra.Command) {
 		config.Mempool.PendingRemoveEvent,
 		"Push event when remove a pending tx",
 	)
+	cmd.Flags().Uint64(
+		"mempool.max_tx_limit_per_peer",
+		config.Mempool.MaxTxLimitPerPeer,
```

Comment:
If we set config.Mempool.MaxTxLimitPerPeer to 0 will never receive any p2p message? Do we need to set 0 to a sepecial meaning? 0 means ulimited?